server/services/import/utils/file.js

```diff
@@ -178,7 +178,7 @@ const getFileTypeChecker = (type) => {
 const getFileDataFromRawUrl = (rawUrl) => {
   const parsedUrl = new URL(decodeURIComponent(rawUrl));
 
-  const fileName = parsedUrl.pathname.split('/').pop().toLowerCase();
+  const fileName = parsedUrl.pathname.toLowerCase().replaceAll('/', '-');
```

Comment:
`replaceAll` caused issue in the past since not all users use a node version ≥ `15`. Prefer using `replace` with a regex.
Also, remove the `http/https.domain.com` part of the url for a cleaner name

Reply:
Pathname doesn't include protocol and domain name.

Reply:
Good point. Just change the `replaceAll` then

Reply:
Done.